### PR TITLE
README changes

### PR DESCRIPTION
**Pull request by Tresquel**
This is how I interpreted #30
* I don't know if how I explained programming in RG is enough but I also don't know what to add
* I don't know if the Roblox documentation is relevant here but it might contain some helpful information so I included it anyway

### CODE REVIEW

File: en/README.md [hunk 1]
```diff
@@ -4,20 +4,28 @@
...
 
-* <a href="https://store.steampowered.com/app/1730260/Retro_Gadgets/" target="_blank">Wishlist the game here!</a>
-* <a href="https://discord.com/invite/np6FUDjt9E" target="_blank">Join the Official Discord Server!</a>
-* [Learn how you can contribute by opening a pull request!](./docs/github/Contributing.md)</a>
+* [Buy the game here!](https://store.steampowered.com/app/1730260/Retro_Gadgets/)
+* [Join the Official Discord Server!](https://discord.com/invite/np6FUDjt9E)
+* [Learn how you can contribute by opening a pull request!](./docs/github/Contributing.md)
+
+### Coding gadgets
+Retro Gadgets uses Luau to code gadgets, which is basically Lua but it [has some differences](https://luau-lang.org/compatibility). You can still use resources for learning Lua and apply that knowledge to Luau. Here are some places to learn Lua (and Luau):
```
Comment (kurmachu) [inline en/README.md:14]: Slight grammatical change, feel free to resolve this chain if you'd like to ignore it.
```suggestion
Retro Gadgets uses Luau to code gadgets, which is basically Lua but [with some differences](https://luau-lang.org/compatibility). You can still use resources for learning Lua and apply that knowledge to Luau. Here are some places to learn Lua (and Luau):
```

**Review by kurmachu — COMMENTED** [review]
Very nice, thanks so much! Just a single suggestion, let me know if you'd like to ignore it and merge anyway.

File: en/README.md [hunk 2]
```diff
@@ -4,20 +4,28 @@
...
 * [Getting Started](./examples/getting-started.md)
 
 ## Todo
-- [ ] Finish README.md
-  - [ ] Add proper description about programming in Retro Gadgets
-  - [ ] Add resources for learning Lua
+- [X] Finish README.md
+  - [X] Add proper description about programming in Retro Gadgets
+  - [X] Add resources for learning Lua
 - [ ] API Documentation
```
Comment (kurmachu) [inline en/README.md:29]: Actually, just noticed, I think this should be checked according to #30 
image
```suggestion
- [X] API Documentation
```

**Comment by Tresquel** [comment]
> Very nice, thanks so much! Just a single suggestion, let me know if you'd like to ignore it and merge anyway.

Will do when I get home 👍

**Review by kurmachu — APPROVED** [review]

**Comment by kurmachu** [comment]
Thanks again! You'll see your changes reflected in the viewer when it syncs.

**Pull request merged**